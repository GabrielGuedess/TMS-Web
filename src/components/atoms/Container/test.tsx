import { render } from '@testing-library/react';

import { Container } from '.';

describe('<Container />', () => {
  it('should render the container correctly', () => {
    const { container } = render(
      <Container>
        <h1>Test</h1>
      </Container>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
