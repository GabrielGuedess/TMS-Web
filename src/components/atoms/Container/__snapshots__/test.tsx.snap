// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render the container correctly 1`] = `
<div
  class="rounded border transition-all border-zumthor-100 bg-white-lilac-50 shadow-default dark:border-shark-950 dark:bg-smoke-950"
>
  <h1>
    Test
  </h1>
</div>
`;
